src/worker/main.ts

```diff
@@ -2,15 +2,15 @@ import { config } from 'dotenv';
 
 config();
 
-import { MoveContactWorker } from './move-contact-worker';
+import { ChtConfWorker } from './cht-conf-worker';
 import { WorkerConfig, checkRedisConnection } from '../config/config-worker';
 
 (async () => {
   const { moveContactQueue, redisConnection} = WorkerConfig;
   await checkRedisConnection();
-  MoveContactWorker.processQueue(
+  ChtConfWorker.processQueue(
     moveContactQueue, 
```

Comment:
`chtConfQueue` ?

Reply:
i think this was addressed when i renamed the queue? is that what you're referring to here?